test/metabase/query_processor_test/cast_test.clj

```diff
@@ -17,11 +17,30 @@
 ;;  * :type/Integer
 ;;  * :type/Number
 
+(defn- ->integer [value]
+  (cond
+    (string? value)
+    (Long/parseLong value)
```

Comment:
What about numbers larger than `Long/MAX_VALUE`? Is it that they aren't supported so we don't test them?